tldraw/tldraw
# Add a toast for missing clipboard permissions.

Shows a toast notifying the user that pasting failed due to clipboard permissions. Resolves https://github.com/orgs/tldraw/projects/53?pane=issue&itemId=70294454

![image](https://github.com/tldraw/tldraw/assets/2523721/704aaca2-a087-4b5b-a0bc-dd710960a318)


### Change type


### Test plan

1. Open a new private window.
2. Copy some shapes and paste them. Make sure you don't allow the clipboard access.
3. You should see the toast.

### Release notes

- Show a toast when pasting failed due to missing clipboard permissions.
